packages/rocketchat-grant/server/index.js

```diff
@@ -12,9 +13,30 @@ import Providers, { middleware as providers } from './providers';
 import Settings from './settings';
 
 let grant;
+let storeAddress = process.env.SESSIONS_URL || 'mongodb://localhost:27017/rocketsession';
+
+if (storeAddress.indexOf('connectTimeoutMS=') < 0) {
+	if (storeAddress.indexOf('?') < 0) {
+		storeAddress += '?connectTimeoutMS=1200';
+	} else {
+		storeAddress += '&connectTimeoutMS=1200';
+	}
+}
+const store = new SessionStorage({
+	collection: process.env.SESSION_COLLECTION || 'mySessions',
+	databaseName: process.env.SESSION_DATABASE || 'connect_mongodb_session',
```

Comment:
same question as before, any reason to store sessions in a different DB other than Rocket.Chat default?

Reply:
sessions won't need to be replicated, don't need them in your backups either, ...